### PR TITLE
Add ruleset: Eustace.io

### PR DESCRIPTION
**Pull request by dnoetzel**

### CODE REVIEW

File: src/chrome/content/rules/Eustace.io.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!--
+	Nonfunctional hosts in *.eustace.io:
+		- www
```
Comment (J0WI) [inline src/chrome/content/rules/Eustace.io.xml]: Please also add `eustace.io` (without `www`).

File: src/chrome/content/rules/Eustace.io.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <!--
 	Nonfunctional hosts in *.eustace.io:
-		- www
+        - eustace.io
```
Comment (J0WI) [inline src/chrome/content/rules/Eustace.io.xml]: Please use tabs to indent lines.

Comment (dnoetzel) [reply src/chrome/content/rules/Eustace.io.xml]: Whoops. I apologize for all of the silly mistakes. Thank you for your patience.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**